File: backend/components/metiers/threed/utils/ai/fallback/ai_fallback.js
// ai_fallback.js – Fallback AI open source pour Threed (threed)
module.exports = {
  aiFallback: (text, lang = 'fr') => {
    // Simule un fallback AI open source pour la traduction ou l'analyse
    if (!text) return '[AI-Fallback] Texte manquant';
    return `[AI-Fallback][${lang.toUpperCase()}] ${text}`;
  }
};

File: backend/components/metiers/threed/utils/ai/fallback/ai_fallback.py
"""
ai_fallback.py – Fallback AI open source pour Threed (threed)
"""


def ai_fallback(text, lang="fr"):
    """
    Simule un fallback AI open source pour la traduction ou l'analyse
    """
    if not text:
        return "[AI-Fallback] Texte manquant"
    return f"[AI-Fallback][{lang.upper()}] {text}"
